src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemConfiguration.tsx

```diff
@@ -369,19 +379,21 @@ const MinimumSize = ({ partition, units }: MinimumSizePropTypes) => {
     <ValidatedTextInput
       ariaLabel="minimum partition size"
       helperText=""
-      validator={() => true}
+      validator={isMountpointMinSizeValid}
       value={convertToDisplayUnits(partition.min_size)}
       type="text"
       onChange={(event, minSize) => {
-        dispatch(
-          changePartitionMinSize({
-            id: partition.id,
-            min_size: convertToBytes(minSize),
-          })
-        );
-        dispatch(
-          changePartitionUnit({ id: partition.id, unit: partition.unit })
-        );
+        if (minSize === '' || /^\d+$/.test(minSize)) {
```

Comment:
With minSize defaulting to 0, the first part of condition can be probably removed?

```suggestion
        if (/^\d+$/.test(minSize)) {
```

Reply:
This is required here, that's what actually transform empty field to 0 :)
But I've removed it from validator :)

Reply:
Ah yeah, I see :sweat_smile: